docs/theme/mlflow/layout.html

```diff
@@ -213,6 +213,8 @@
                 const message = document.createElement('span');
                 message.textContent = "Code copied to clipboard!";
```

Comment:
```suggestion
                message.textContent = "Copied";
```

Reply:
I like this :) adjusting the CSS for the sizing. After I verify the spacing, I'll merge this PR.